config-connector/solutions/iam/helm/member-iam/values.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+projectID:
+
+iamPolicyMember:
+  iamMember: member-iam-test
+
+role: roles/compute.networkAdmin
```

Comment:
Do you think it's better to also move "role" to be under "iamPolicyMember" just like what you did for project-iam solution?